webrtc.html

```diff
@@ -4218,6 +4218,8 @@
               </tbody>
             </table>
           </div>
+          <p>Note: the user agent will typically only gather <code>active</code> 
+          ICE TCP candidates.</p>
```

Comment:
FWIW ... I don't really know what that means. Could we point at some other docs to help explain